invenio_communities/views/communities.py

```diff
@@ -150,6 +150,11 @@ def communities_new():
     can_create = current_communities.service.check_permission(g.identity, "create")
     if not can_create:
         raise PermissionDeniedError()
+
+    can_create_restricted = current_communities.service.check_permission(
```

Comment:
minor: shouldn't this be just `can_manage_access` again? I know it's not very explicit, but do we want to actually separate the two actions? I.e. is there a scenario where we allow a user to create restricted communities, but not change the visibility afterwards?